app/src/main/java/com/example/footballapp/presentation/custom_view/TeamActionsCustomView.kt

```diff
@@ -0,0 +1,166 @@
+package com.example.footballapp.presentation.custom_view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.view.isInvisible
+import com.example.footballapp.R
+import com.example.footballapp.databinding.TeamActionsCustomViewBinding
+import com.example.footballapp.presentation.ui.matches.types.GoalType
+import com.example.footballapp.presentation.ui.matches.types.MatchActionsType
+import com.example.footballapp.presentation.ui.matches.types.MatchTeamType
+import com.example.footballapp.util.extensions.isVisible
+import com.example.footballapp.util.extensions.setBackground
+import com.example.footballapp.util.extensions.setBackgroundTint
+import com.example.footballapp.util.getActionType
+
+class TeamActionsCustomView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet,
+    defStyleAttrs: Int = 0,
+) : ConstraintLayout(context, attrs, defStyleAttrs) {
+
```

Comment:
აუტო გოლზე რო წითლად უნდა დაეწეროს რაღაც მაგის ლოგიკასაც ვერ ვხედავ

Reply:
კაი აქ თვალს დავხუჭავ უცებ ჩავმერჯოთ ეს რო  იმ ბრენჩამდე მივიდეთ დღეს რაც გვანახე  და იქ დაგაყრი ერთიანად , აქ ეხლა დიდი პრობლემა არაა მარა ზოგადად უშუალოდ სფეისის პროექტთან რო გექნება შეხება ესე პრები ერთმანეთს არ გაასწრო პრობლემები შეგექმნება

Reply:
solved that problem